collectors/bzimport/convertors.py

```diff
@@ -976,26 +987,50 @@ def bug2flaws(self):
                 )
             ]
 
-        #############
-        # CVE flaws #
-        #############
+        ###################
+        # single-CVE flaw #
+        ###################
+
+        if len(self.cve_ids) == 1:
+            logger.debug(f"{self.__class__}: processing {self.cve_ids[0]}")
+
+            # if there was multiple flaws but now it is only one
+            # we remove all the previous as the mapping is ambiguous
+            # except the CVE itself where the mapping is straightforward
+            if existing_flaws.count() > 1:
+                existing_flaws.exclude(cve_id__in=self.cve_ids).delete()
```

Comment:
since we know there's only one item in `self.cve_ids` it's probably much cheaper to convert this to:

```python
existing_flaws.get(cve_id=self.cve_ids[0]).delete()
```

Reply:
since you'll be using `self.cve_ids[0]` multiple times you might want to assign a variable to it too

Reply:
my bad, the `exclude` remains but probably still cheaper to use direct comparison than the `IN` operator

Reply:
I will replace the IN operator